test/extensions/ERC721ALowCap.test.js

```diff
@@ -0,0 +1,58 @@
+const { expect } = require('chai');
+
+describe('ERC721ALowCap', function () {
+  beforeEach(async function () {
+    this.ERC721ALowCap = await ethers.getContractFactory('ERC721ALowCapMock');
+    this.token = await this.ERC721ALowCap.deploy('Azuki', 'AZUKI');
+    await this.token.deployed();
+  });
+
+  context('with minted tokens', async function() {
```

Comment:
Use 2 spaces for indentation in this context instead of 4.

Reply:
Should be able to run `npm run lint` to autofix all formatting issues!